views/index.ejs

```diff
@@ -79,5 +66,32 @@
   </form>
   <iframe id="emulator" class="emulator" scrolling="no"></iframe>
 </div>
+
+<script>
+  function emulate(mode) {
+    let url = '/latest'
+    const urls = {newspapers: '?papers=', devices: '?deviceId='}
+    for (const i in urls) {
+      if (mode === i) {
+        const items = $(`input[name=${i}]:checked`).map(function () {return this.value}).get()
+        if (items.length) url += urls[i] + items.join(',')
+      } else {
+        $(`input[name=${i}]`).prop('checked', false)
+      }
+    }
+    $('#url').html(`<a href="${url}" target="_blank">${window.location.protocol + '//' + window.location.host + url}</a>`)
+    $('#emulator').prop('src', url)
+  }
+
+  // initialize emulation on load
+  const db = <%-JSON.stringify(db)%>;
+
+  emulate(
+    // if there is an `isSelected` newspaper, initialize in newspapers mode, otherwise initialize in devices mode
```

Comment:
Perhaps move this comment to `db.js`?

Reply:
Kept this comment here, and also added a further comment in `db.js`.